core/domain/obj_services.py

```diff
@@ -44,10 +44,8 @@ def _refresh_registry(cls):
 
             ancestor_names = [
                 base_class.__name__ for base_class in inspect.getmro(clazz)]
-            if 'BaseObject' not in ancestor_names:
-                continue
-
-            cls.objects_dict[clazz.__name__] = clazz
+            if 'BaseObject' in ancestor_names:
```

Comment:
I understand that you're trying to simplify the code here, but I would prefer to have an explicit "exit early" step for clarity. Let's leave the original code as-is.

Reply:
Actually I can not seem to test the continue statement that's why I rearranged the code

Reply:
Done!